exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+
+  <div id="perseus">
+    <div>
+      <div id="problem-area">
+        <div id="workarea"></div>
+        <div id="hintsarea"></div>
+      </div>
+      <div id="answer-area-wrap">
+          <div id="answer-area">
+              <div class="info-box">
+                  <div id="solutionarea"></div>
+                  <button @click="checkAnswer" v-if="!correct" id="check-answer-button">{{ checkText }}</button>
+                  <button v-else id="next-question-button">{{ $tr('correct') }}</button>
```

Comment:
the linter won't complain about the single quotation mark here around the 'correct'?